l10n_ch_hr_payroll/models/hr_payroll_config.py

```diff
@@ -42,6 +43,14 @@ def _get_default_net(self):
         all_equal = self.search_account_by_rule([
             ('l10n_ch_hr_payroll.NET_CH', 'credit')])
 
+        return all_equal\
+
+    @api.model
+    def _get_default_provision13(self):
+        all_equal = False
+        all_equal = self.search_account_by_rule([
```

Comment:
You can directly return the result instead of storing it in a variable. `return self.search_account_by_rule([`

Reply:
```suggestion
        return self.search_account_by_rule([
```